ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -875,6 +875,23 @@ class JITModBuilder {
     if (OldGV->isConstant()) {
       GV->copyAttributesFrom(&*OldGV);
       cloned_globals.push_back(OldGV);
+      // If this is a vector containing other constants, we need to clone
+      // those as well.
+      Type *Ty = OldGV->getValueType();
+      Constant *C = OldGV->getInitializer();
+      if (Ty->isArrayTy()) {
+        if ((cast<ArrayType>(Ty))->getElementType()->isPointerTy()) {
+          auto GVA = dyn_cast<ConstantArray>(C);
+          for (size_t I = 0; I < Ty->getArrayNumElements(); I++) {
+            auto Elem = GVA->getAggregateElement(I);
+            if (isa<GlobalVariable>(Elem)) {
```

Comment:
We can fold these two `if`s together.

Reply:
[fd302af](https://github.com/ykjit/yk/pull/618/commits/fd302aff8d6e40b6e04dbf92b966ee67583b6ac7)